app/presenters/return-requirements/remove.presenter.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+/**
+ * Formats data for the `/return-requirements/{sessionId}/remove/{requirementIndex}` page
+ * @module RemovePresenter
+*/
+
+const { formatLongDate } = require('../base.presenter.js')
+
+/**
+ * Formats data for the `/return-requirements/{sessionId}/remove/{requirementIndex}` page
+ *
+ * @param {module:SessionModel} session - The returns requirements session instance
+ * @param {string} requirementIndex - The index of the requirement being added or changed
+ *
+ * @returns {Object} - The data formatted for the view template
+ */
+function go (session, requirementIndex) {
+  const { id: sessionId, licence, requirements } = session
+  const requirement = requirements[requirementIndex]
+
+  return {
+    backLink: _backLink(session),
+    licenceId: licence.id,
+    licenceRef: licence.licenceRef,
+    returnRequirement: _formattedReturnRequirement(requirement) ? _formattedReturnRequirement(requirement) : null,
```

Comment:
I've suggested a change which means you should never get `null` back from the function. But FYI, the function was determining whether null or a value was being returned. There wasn't any need for the ternary here! 😁 

```suggestion
    returnRequirement: _formattedReturnRequirement(requirement),
```